kmp-library/src/jvmMain/kotlin/uk/co/lucasweb/aws/v4/signer/hash/StringExtension.kt

```diff
@@ -0,0 +1,3 @@
+package uk.co.lucasweb.aws.v4.signer.hash
+
+actual fun String.toUtf8ByteArray(): ByteArray = toByteArray()
```

Comment:
`String. toByteArray()` just exists on the jvm and relies on `java.lang.StringCoding`, so we have to provide a different implementation for `native`.

Reply:
What about using [`.toUtf8`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.native/to-utf8.html) here? Should be available for common modules too

Reply:
Unfortunately, it's not. If you follow the link you posted and just enable `common`, you will see that there is nothing :(

Reply:
My bad, sorry